include/queue/queue_sycl.hpp

```diff
@@ -26,7 +26,7 @@
 #define QUEUE_SYCL_HPP
 
 #include <CL/sycl.hpp>
-#include <queue/pointer_mapper.hpp>
+#include <computecpp-sdk/include/vptr/virtual_ptr.hpp>
```

Comment:
If you set the include directory for your targets, you'll be able to do vptr/virtual_pointer.hpp which I think is much nicer.